### PR TITLE
control-service: Python image based on Photon OS

### PR DESCRIPTION
**Pull request by mivanov1988**
# Why
We don't have a replacement for `python:slim` image with installed dependencies needed for the installation of VDK SDK (e.g. quickstart-vdk).

# What
I added an additional image called `python-$PYTHON_MAJOR.$PYTHON_MINOR-secure` which includes dependencies needed for the installation of VDK SDK. Both VDK SDK and base job image will be based on it, in order to be compliant.

# Testing Done
https://gitlab.com/vmware-analytics/versatile-data-kit/-/jobs/4482856767

Note: We will provide separate CI/CD steps in a separate PR.

Signed-off-by: miroslavi@vmware.com

### CODE REVIEW

File: projects/control-service/projects/job-base-image-secure/publish-job-base.sh [hunk 1]
```diff
@@ -11,23 +11,36 @@ function build_and_push_image() {
...
+    data_job_base_name="data-job-base-python-$PYTHON_MAJOR.$PYTHON_MINOR-secure"
+    python_docker_file="Dockerfile-python"
+    data_job_base_docker_file="Dockerfile-data-job-base"
 
-    image_repo="$VDK_DOCKER_REGISTRY_URL/$name"
-    image_tag_local="$image_repo:local"
-    image_tag_version="$image_repo:$VERSION_TAG"
-    image_tag_latest="$image_repo:latest"
+    python_image_repo="$VDK_DOCKER_REGISTRY_URL/$python_name"
+    python_image_tag_local="$python_image_repo:local"
```
Comment (tozka) [inline projects/control-service/projects/job-base-image-secure/publish-job-base.sh:20]: python_image_tag_local doesn't seem used

File: projects/control-service/projects/job-base-image-secure/Dockerfile-data-job-base [hunk 2]
```diff
@@ -1,96 +1,9 @@
...
-      ${_pylibdir}/sqlite3/test \
-      ${_pylibdir}/test \
-      ${_pylibdir}/tkinter/test \
-      ${_pylibdir}/unittest/test \
- && find ${_pylibdir} -type d -name __pycache__ -exec rm -rf '{}' +
-
-FROM photon:latest as base-job-image
-COPY --from=builder /usr/local/ /usr/local/
+FROM photon:latest
+COPY --from=base /usr/local/ /usr/local/
```
Comment (tozka) [inline projects/control-service/projects/job-base-image-secure/Dockerfile-data-job-base:6]: Consider adding comments in the Dockerfiles to provide some context or explanation . 

In this cause what is in /usr/local  and why are we copying it. What if it does not exists

**Review by tozka — APPROVED** [review]
Looks good to me. 

I made some comments but they are not blocking. Feel free to address them as you see fit and merge.

**Review by dakodakov — APPROVED** [review]

File: projects/control-service/projects/job-base-image-secure/publish-job-base.sh (hunk 1 above)
Comment (mivanov1988) [reply projects/control-service/projects/job-base-image-secure/publish-job-base.sh:20]: Correct

File: projects/control-service/projects/job-base-image-secure/Dockerfile-data-job-base (hunk 2 above)
Comment (mivanov1988) [reply projects/control-service/projects/job-base-image-secure/Dockerfile-data-job-base:9]: Added

**Pull request merged**